### ISSUE TITLE
Label width auto does not update its actual width in ListView

### ISSUE THREAD

**Issue by PeterStaev** [NativeScript/NativeScript#5506]
### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
Yes

### Tell us about the problem
If you have an `itemTemplate` with a `Label` that is for example set in a `GridLayout` column which width is set to `auto`, when the view is reused and the other data item bound text is longer than what was initially measured, the width of the `Label` does not get updated, so the longer text gets cut. 

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 3.4
- Cross-platform modules: 3.4.0
- Runtime(s): 3.4.2 (android)

### Please tell us how to recreate the issue in as much detail as possible.
https://play.nativescript.org/?template=play-tsc&id=Q1xikN&v=4
Scroll down to the bottom of the list on android and you will see that the numbers on the right are missing.

**Comment by tsonevn** [comment]
Hi @PeterStaev,
Thank you for the attached sample project.
I tested the case and indeed this is a real issue, which is reproducible only on Android. 
Meanwhile, I would suggest replacing the auto option with * (Star), which will allow rendering the whole text.
```
<ListView items="{{ items }}">
		<ListView.itemTemplate>
			<GridLayout columns="*, *">
				<Label text="Item" col="0" />
				<Label text="{{ number }}" col="1" />
			</GridLayout>
		</ListView.itemTemplate>
	</ListView>
```

**Comment by jogboms** [comment]
@PeterStaev I had to use a suggested workaround by wrapping the `Label` within a `FlexboxLayout`.

**Comment by PeterStaev** [comment]
Sadly flexbox does not help if you have a list, tap on item to edit it, save & go back to the list. If you changed data that affects the number you are stuck with using whatever the width of the previous item was... Really annoying problem 😞

**Comment by vakrilov** [comment]
Good news - this seems to be resolved in `tns-core-modules@4.2.0`. 
Might be related to this PR: https://github.com/NativeScript/NativeScript/pull/5917
